src/pleasant-lcd.h

```diff
@@ -145,6 +145,8 @@ enum lcd_orientation {
                          | LCD_MEMORY_ACCESS_CONTROL_MY),
 };
 
+enum lcd_orientation lcd_current_orientation;
+
```

Comment:
Move this to the State block below, and make it `extern` like everything there. Then initialize it to a sane value at the top of `pleasant-lcd.c`